### ISSUE TITLE
Support for Named Pipe connection

### ISSUE THREAD

**Issue by robustsoftware** [microsoft/mssql-jdbc#1150]
Support for a named pipe connection, so that java application can connect to LocalDB instance.

**Comment by peterbae** [comment]
Hi @robustsoftware, support for named pipes is in our backlog but there is no timeline for it at this time. You can refer to #769 for the discussion that took before.

**Comment by David-Engel** [comment]
> Any timing on this issue?

No. This item is still in the backlog but has no timeline associated with it.

Regards,
David

**Comment by bardware** [comment]
> It's only possible with a fork of jTDS which is not available on any maven repository

Someone put a jar file on clojars
https://github.com/milesibastos/jTDS/issues/5#issuecomment-678006455
https://clojars.org/net.sourceforge.jmarcos00/jtds

**Comment by snowinmars** [comment]
Ping for fix >:3
